cross-media-measurement/src/main/proto/wfa/measurement/common/crypto/liquid_legions_v2_encryption_methods.proto

```diff
@@ -50,36 +50,36 @@ message RegisterNoiseGenerationParameters {
   ElGamalPublicKey composite_el_gamal_public_key = 1;
   // The elliptical curve to work on.
   int64 curve_id = 2;
-  // The total number of noise registers to be added.
-  // The value should be at least equal to
-  //   publisher_noise_parameters.offset * 2
-  //   + global_reach_dp_noise_parameters.offset * 2
-  //   + blind_histogram_noise_parameters.offset * total_sketches_count
-  //   * (total_sketches_count + 1).
-  // TODO: add link to the external paper.
-  int32 total_noise_registers_count = 3;
+  // Number of contributors to the noise.
+  int32 contributors_count = 3;
```

Comment:
I find the naming to be a bit hard to understand. Maybe we should have a more detailed comments (i.e. saying something like this is the number of honest parties contributing their noise). Wdyt?

(Same comments for similar naming below.)

Reply:
sg. Done